resources/assets/less/components/comment.less

```diff
@@ -1,3 +1,52 @@
+.comment-form {
+
+    margin-bottom: 1em;
+
+    textarea {
+        width: 100%;
```

Comment:
Again, personal picky-ness: Can you alphabetize the properties in all the less sections? e.g. go from `border` to `width`